central/main.go

```diff
@@ -681,6 +681,12 @@ func customRoutes() (customRoutes []routes.CustomRoute) {
 		{
 			Route:         "/api/docs/swagger",
 			Authorizer:    user.Authenticated(),
+			ServerHandler: docs.SwaggerV2(),
```

Comment:
The existing API should be handled by the existing handler.

Reply:
Ah I messed up ordering. Fixed it.